src/workshops/applicant-git-2/index.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Git introduction
+description: This workshop is an introduction to using Git for version control; GitHub for hosting a codebase and deploying a website; and the terminal for creating files and moving between directories.
+tags:
+  - workshop
+  - git
+keywords:
+  - git
+  - introduction
+starter: false
+---
+
+This workshop will help you practise **Git flow**. Follow these steps to connect a local repository to a remote one on GitHub. Before following these steps, you'll need to [install Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git).
+
+This workshop is an advancement on concepts covered in [our Git introduction](../applicant-git-1/). Completing that workshop is not a requirement to understand this one, however you might prefer to focus on the fundamentals there before diving into this workshop.
+
+This workshop assumes you have a working knowledge of [using the terminal](../terminal-intro/).
+
+## Local repository
+
+Create a folder on your computer and add files for HTML, CSS and JavaScript. Add some content to the HTML, link the CSS and add a script tag which points to your JS.
+
+Open up your terminal and navigate to the folder where you've saved your files. Initialise this folder as a Git repository. You can do this by running the command `git init` in the working directory.
+
+## Staging and committing
+
+Staging tells Git which files we'd like to add to our next commit. When you stage changes you can continue to make changes in the working directory and control which files or changes you'd like to include in a commit.
+
+When staging, you can specify files to stage, or using the `.` you can stage all the files in your current directory.
+
+```
+git add index.html
+git add .
+```
+
+Commiting is like saving your progress or taking a snapshot of the current codebase. You are telling Git that all the changes made should be tracked. The repository will be saved at the point in time you commit.
+
+Running the command without the `-m` flag will open up an editor in your terminal where you can write a commit message. Exit this by hitting `esc` and typing `:wq`.
```

Comment:
Is it worth mentioning here that they can [change their default terminal editor](https://oliverjam.es/blog/make-your-terminal-nicer/#set-your-default-editor) to avoid Vim opening every time they commit?

It's quite nice just typing `git commit` and having VS Code open